components/studentPortal/layout/Sidebar.tsx

```diff
@@ -96,22 +121,46 @@ export const Sidebar: React.FC = () => {
     //Full width then restrict in page
     <div className="flex flex-col w-full bg-backgroundPrimary text-textPrimary">
       <div className="grid">
-        <div className="flex p-4">
-          {user && (
+        {user && userRole === "freemium" ? (
+          <div className="flex items-center p-4">
```

Comment:
I might extract this out into a component called `FreemiumSidebarHeader`

Reply:
great idea.  Created a freemium folder in the studentPortal section of components and put it there.
makes the code a lot cleaner in the sidebar.